weaviate/embedded.py

```diff
@@ -29,8 +29,8 @@
 class EmbeddedOptions:
     persistence_data_path: str = os.environ.get("XDG_DATA_HOME", DEFAULT_PERSISTENCE_DATA_PATH)
     binary_path: str = os.environ.get("XDG_CACHE_HOME", DEFAULT_BINARY_PATH)
-    version: str = "1.19.12"
-    port: int = 6666
+    version: str = "latest"
```

Comment:
please reverse this changes, simply using "latest" does not work: https://github.com/weaviate/weaviate-python-client/issues/350